common/tags/src/test/java/org/apache/spark/tags/ExtendedSQLCommandTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.tags;
+
+import org.scalatest.TagAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@TagAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+public @interface ExtendedSQLCommandTest { }
```

Comment:
Do you have a more recommended name? So that we can add more cases to this group

Reply:
@zhengruifeng 
1.  Do you have any suggestions for the name of the tag?
2.  I am thinking of moving what type of test to this new group. One way is to have the same type of feature, such as now moving `Command` related cases to this group, and the other way is to move the first x% (possibly 30%?) of the slowest running cases to this group. Which do you think is more reasonable? Do you have any better suggestions?

Reply:
> Do you have any suggestions for the name of the new tag?

I feel `ExtendedSQLCommandTest` suggest only sql commands related tests can be used with this tag. 


> I am thinking of moving what type of test to this new group. One way is to have the same type of feature, such as now moving Command related cases to this group, and the other way is to move the first x% (possibly 30%?) of the slowest running cases to this group. Which do you think is more reasonable? Do you have any better suggestions?

I think we can just find the slowest tests, don't need to be the same group.

also cc @HyukjinKwon @dongjoon-hyun

Reply:
Yes, I'm thinking of a more universal name, but I don't have any good ideas yet